client/app/services/editor.service.js

```diff
@@ -18,6 +18,7 @@
             launchFieldPapersEditor: launchFieldPapersEditor,
             launchPotlatch2Editor: launchPotlatch2Editor,
             launchIdEditor: launchIdEditor,
+            launchIdX: launchIdX,
```

Comment:
Here too :)

Reply:
Argggghhhh. Search and Replace has failed me miserably

Reply:
Fixed 23c4bb4